src/CMakeLists.txt

```diff
@@ -20,9 +20,12 @@ cmake_minimum_required(VERSION 3.6)
 project(eapis C CXX)
 
 include(${SOURCE_CMAKE_DIR}/project.cmake)
-init_project()
+init_project(
+    INCLUDES ../include

```

Comment:
I think that manually adding this include folder isn't necessary, since, if I remember it right, `init_project` adds it by default.

Reply:
In this patch, I removed that as I'm worried about a <path>/../include accidentally including files it shouldn't be as it is blindly including a folder outside of its self.

Reply:
that is in this PR: https://github.com/Bareflank/hypervisor/pull/581

Reply:
Ah, ok.